airflow/contrib/operators/gcp_function_operator.py

```diff
@@ -20,268 +20,14 @@
 
 from googleapiclient.errors import HttpError
 
-from airflow import AirflowException, LoggingMixin
+from airflow import AirflowException
```

Comment:
Note to self - we should separate extraction of the Field Validator to a separate commit.

Reply:
Abandoned. We do not need to split it.